docs/community/faqs.md

```diff
@@ -1,34 +1,42 @@
 ---
 id: faqs
-title: 'Frequently asked Questions'
-sidebar_label: 'FAQS'
+title: 'Frequently Asked Questions'
+sidebar_label: FAQs
 keywords:
   - faqs
 ---
 
+## Contributions
+
 1. **How do I find good first issues?**
```

Comment:
Do we want to keep the `ol` as it is or make them `h3` for better read?

![ol vs h3](https://github.com/open-sauced/docs/assets/45172775/81f45164-d9c9-4c8f-9fa1-8f17026508a8)